azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/state/ParticipantsState.kt

```diff
@@ -7,5 +7,6 @@ import com.azure.android.communication.ui.chat.models.RemoteParticipantInfoModel
 
 internal data class ParticipantsState(
     val participants: Map<String, RemoteParticipantInfoModel>,
-    val participantTyping: Set<String>
+    val participantTyping: Set<String>,
+    val participantsListVisible: Boolean,
```

Comment:
Should the ParticipantState hold onto a value for whether or not the list is visible? 
This seems incorrect to me. Whether or not the participant view is visible is a view concept and has nothing to do with the Participants state. @ahammer  @iaulakh  thoughts?